packages/firestore/src/platform_browser/browser_platform.ts

```diff
@@ -72,4 +73,27 @@ export class BrowserPlatform implements Platform {
   btoa(raw: string): string {
     return btoa(raw);
   }
+
+  randomBytes(nBytes: number): Uint8Array {
+    if (nBytes <= 0) {
+      return new Uint8Array();
+    }
+
+    // Polyfill for IE and WebWorker
+    // eslint-disable-next-line no-restricted-globals
+    const crypto =
```

Comment:
Should we add support for "self" as well? 
https://stackoverflow.com/questions/11219775/global-variable-in-web-worker

We might also consider adding support for this shim on ReactNative: https://stackoverflow.com/questions/45301900/howto-patch-shim-crypto-getrandomvalues-for-react-native

While the fallback will make sure these environments don't break, we can possibly extend our support for `crypto`.

Reply:
So we don't need it, right?

Reply:
I am ok with not adding support for ReactNative until the ReactNative Firebase wrapper bundles "crypto", but note that without getting "crypto" from globals, we will always fall back to Math.random() for RN.

Reply:
Yep, I am filing a ticket to get crypto added for react native firestore.

Reply:
react-native-firebase participant here, just cross-linking https://github.com/invertase/react-native-firebase/issues/3618

Reply:
Just thought I'd quickly summerize; React Native Firebase doesn't use the JS SDK, we're a thin JS wrapper around the native Android & iOS SDKs so this change has no direct affect on us. 

However our JS wrapper does use an identical random id generator - so I think we may need to port this change across in some form - we'll pick this up separately, thanks!